xla/service/gpu/gpu_latency_hiding_scheduler.cc

```diff
@@ -426,7 +441,15 @@ int64_t GpuAsyncTracker::GetNumAvailableResources(int64_t resource_type) const {
   }
 
   if (resource_type ==
-      ResourceTypeToIndex(GpuResourceType::kGpuAsyncStreamCollectivesP2P)) {
+          ResourceTypeToIndex(GpuResourceType::kGpuAsyncStreamCollectivesP2P) ||
+      resource_type ==
+          ResourceTypeToIndex(GpuResourceType::kGpuAsyncStreamSend0) ||
```

Comment:
Can you provide more context for this change?

Reply:
this is to fix those send recv tests under gpu_latency_hiding_scheduler_test, they used to be hidden by the scheduling_instruction_crosses_overlap_limit, since we are disabling the overlap limit function in this pr, they start failing due to exceeding resource limit. The resource is not given correctly for pipelined send recvs.